include/industry_standard/spdm.h

```diff
@@ -199,7 +210,9 @@ typedef struct {
     /* param2 == RSVD*/
     uint16_t length;
     uint8_t measurement_specification;
-    uint8_t reserved;
+    /* other_params_support is added in 1.2.*/
+    /* BIT[0:3]=opaque_data_format support, BIT[4:7]=reserved*/
+    uint8_t other_params_support;
```

Comment:
Ugh, the name of the field is officially OtherParamsSupport? Had I been paying more attention to 1.2 I would have renamed it.